easy-ui-react/src/Tabs/useIndicatorSizing.ts

```diff
@@ -1,30 +1,30 @@
 import { useResizeObserver } from "@react-aria/utils";
 import { MutableRefObject, useCallback, useEffect } from "react";
-import { TabNavContextType } from "./context";
+import type { TabsContextType } from "./context";
```

Comment:
**question**: is explicit type import intentional?

Reply:
it was intentional, to guard against a circular dependency import before i extracted out this hook into its own file. it's not necessary anymore though so i removed it